modules/user/src/main/LightUserApi.scala

```diff
@@ -64,7 +73,9 @@ final class LightUserApi(repo: UserRepo, cacheApi: CacheApi)(using Executor) ext
             name = name,
             title = doc.getAsOpt[chess.PlayerTitle](F.title),
             flair = doc.getAsOpt[Flair](F.flair).filter(FlairApi.exists),
-            isPatron = ~doc.child(F.plan).flatMap(_.getAsOpt[Boolean]("active"))
+            isPatron = ~doc.child(F.plan).flatMap(_.getAsOpt[Boolean]("active")),
+            disabled = doc.getAsOpt[Boolean](F.enabled).fold(true)(!_),
+            tosViolation = doc.getAsOpt[UserMarks](F.marks).fold(false)(m => m.boost || m.engine)
```

Comment:
Not a fan of just copy-pasting what is already done in [user](https://github.com/lichess-org/lila/blob/a4697635de4550acd54fcdb6b725fe7ce32ff2a5/modules/user/src/main/User.scala#L65) since it adds extra maintenance overhead. Open for suggestions.